tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>os.environ["CUDA_DEVICE_ORDER"]="PCI_BUS_ID"</p><p>os.environ["CUDA_VISIBLE_DEVICES"]="1"</p><p> </p><p>If this doesn't make things more difficult to iterate, I'd remove it now. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>